### ISSUE TITLE
Landing page on southplattebasin.com needs to call out stories better

### ISSUE THREAD

**Issue by smalers** [OpenWaterFoundation/swsi-story-sp-entities#29]
From Megan Holcolm, 2018-11-27:

Some quick feedback on the Story Map landing page here. While the three stories are linked in the landing page text, it is not super clear that there are three different stories! 
For those who skim web text/dont know what they are looking for, it might be helpful to create 3 bullet points (in addition to the inline links) that clearly say:
"Story 1: _title_;
Story 2: _title_;
Story 3: _title_"

What do you think?

**Comment by smalers** [comment]
I emailed Megan:

I am copying Lacey Williams, who has coordinated changes on the southplatebasin.com website so she can consider your feedback. OWF did not actually edit that landing page but I agree that using links in narrative makes them harder to distinguish and find the links.  Since we have completed the project and don't have a role in updates to the BRT website, changes need to be coordinated through Lacey.  

[On the OWF stories website where we do initial deployment testing](http://stories.openwaterfoundation.org/), we have structured many of our main landing pages to have a standard format that hopefully clearly identifies each story.  The BRT website maintainers may or may not have the skills to do something like this but should be able to format bulleted lists and other basic HTML formatting.
